src/Compilers/CSharp/Test/Syntax/LexicalAndXml/LexicalErrorTests.cs

```diff
@@ -573,48 +573,48 @@ public static int Main ()
         }
 
         [Fact]
-        public void CS1035AtColonParsedAsComment_01()
+        public void CS1035AtColonParsedAsBadRazorContent_01()
         {
             var test = """
                 var x = @:;
                 """;
 
             ParsingTests.ParseAndValidate(test,
-                // (1,9): error CS1056: Unexpected character '@'
+                // (1,9): error CS1525: Invalid expression term ''
                 // var x = @:;
-                Diagnostic(ErrorCode.ERR_UnexpectedCharacter, "@").WithArguments("@").WithLocation(1, 9),
-                // (1,12): error CS1733: Expected expression
+                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "@:;").WithArguments("").WithLocation(1, 9),
+                // (1,10): error CS1646: Keyword, identifier, or string expected after verbatim specifier: @
                 // var x = @:;
-                Diagnostic(ErrorCode.ERR_ExpressionExpected, "").WithLocation(1, 12),
+                Diagnostic(ErrorCode.ERR_ExpectedVerbatimLiteral, ":").WithLocation(1, 10),
                 // (1,12): error CS1002: ; expected
                 // var x = @:;
                 Diagnostic(ErrorCode.ERR_SemicolonExpected, "").WithLocation(1, 12));
         }
 
         [Fact]
-        public void CS1035AtColonParsedAsComment_02()
+        public void CS1035AtColonParsedAsBadRazorContent_02()
         {
             var test = """
                 @:<div>test</div>
                 """;
 
             ParsingTests.ParseAndValidate(test,
-                // (1,1): error CS1056: Unexpected character '@'
+                // (1,2): error CS1646: Keyword, identifier, or string expected after verbatim specifier: @
                 // @:<div>test</div>
-                Diagnostic(ErrorCode.ERR_UnexpectedCharacter, "@").WithArguments("@").WithLocation(1, 1));
+                Diagnostic(ErrorCode.ERR_ExpectedVerbatimLiteral, ":").WithLocation(1, 2));
         }
 
         [Fact]
-        public void CS1035AtColonParsedAsComment_03()
+        public void CS1035AtColonParsedAsBadRazorContent_03()
         {
             var test = """
                 @: M() {}
```

Comment:
>M() {} [](http://example.com/codeflow?start=19&length=6)

Consider verifying this C# code is actually treated as trivia.

Reply:
It's not treated as trivia, it's part of the token content.

Reply:
> It's not treated as trivia, it's part of the token content.

Ah right, thanks. My original comment was really about verifying this was treated as part of the razor content token rather than parsed separately as C#.

Reply:
Is this not covered by the test changes in `LexicalTests`?

Reply:
Yes, `LexicalTests` appear to cover this.